### ISSUE TITLE
Convolution and dense layers default initializer 

### ISSUE THREAD

**Issue by saeedalahmari3** [keras-team/keras#13995]
In Keras documentation, glorot_uniform says that the initializer is using Glorot Uniform from [this paper](http://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf). However, the Keras implementation is totally different from the equation on the paper. Also, there are some arguments such as mode ='fan_avg' is the default. It should be same as the referenced paper. 'fan_sum'. Golort uniform is shown in Equation 1, but Keras implementation is shown in Equation 2. I had hard time to produce the same or close results to Keras using Pytorch. 
![Screen Shot 2020-04-23 at 9 44 11 AM](https://user-images.githubusercontent.com/23180823/80107996-ce642d80-8549-11ea-9bc9-a45262ac1f60.png)

`def __call__(self, shape, dtype=None):
        fan_in, fan_out = _compute_fans(shape)
        scale = self.scale
        if self.mode == 'fan_in':
            scale /= max(1., fan_in)
        elif self.mode == 'fan_out':
            scale /= max(1., fan_out)
        else:
            scale /= max(1., float(fan_in + fan_out) / 2)
        if self.distribution == 'normal':
            # 0.879... = scipy.stats.truncnorm.std(a=-2, b=2, loc=0., scale=1.)
            stddev = np.sqrt(scale) / .87962566103423978
            x = K.truncated_normal(shape, 0., stddev,
                                   dtype=dtype, seed=self.seed)
        else:
            limit = np.sqrt(3. * scale)
            x = K.random_uniform(shape, -limit, limit,
                                 dtype=dtype, seed=self.seed)
        if self.seed is not None:
            self.seed += 1
        return x`

**Comment by TAMuga-arch** [comment]
Suppose you have a sentence x which is a string; e.g., x = "erath is falt".
Write the Python code for taking a sentence x and producing a dict representing the following two feature templates please help to solve

**Comment by Saduf2019** [comment]
@saeedalahmari3 
Moving this issue to closed status as there has been no activity, in case you still face the error please create a new issue.
